src/components/FlatParameters/FlatParameters.js

```diff
@@ -290,15 +312,25 @@ export const FlatParameters = ({
                                 ));
                             }}
                             renderInput={(inputProps) => (
-                                <TextField {...inputProps} variant={variant} />
+                                <TextField
+                                    {...inputProps}
+                                    variant={variant}
+                                    sx={{
+                                        ...(showSeparator && {
+                                            input: {
+                                                height: '10px',
+                                            },
+                                        }),
+                                    }}
+                                />
                             )}
                         />
                     );
                 } else {
                     // no possible values => free user inputs
                     return (
                         <Autocomplete
-                            fullWidth
+                            fullWidth={!showSeparator}
```

Comment:
Why the width of the Autocomplete depends of the prop `showSeparator` ?

Reply:
It depends on if it's on specefic parameters or in useImportExport